install.sh

```diff
@@ -26,6 +21,7 @@ elif [[ $(uname) == "Darwin" ]]; then
     MINICONDA_OS="MacOSX"
     MINICONDA_ARCH=$(uname -m)
     PYTHON_VERSION="3.9"
+    echo "MacOSX only supported with python 3.9"
```

Comment:
Lets produce this message only when PYTHON_VERSION is not 3.9 and is forced to 3.9